src/Engine/Manager/ScriptManager.cpp

```diff
@@ -703,22 +686,22 @@ void ScriptManager::ExecuteScriptMethod(const Entity* entity, const std::string&
 void ScriptManager::CreateInstance(Component::Script* script) {
     currentEntity = script->entity;
     ScriptFile* scriptFile = script->scriptFile;
-    
+
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The function 'ExecuteScriptMethod' is never used. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AunusedFunction)